pkg/packagekit/updates.jsx

```diff
@@ -305,6 +315,8 @@ class UpdateItem extends React.Component {
                                 <dd>{pkgs}</dd>
                                 { cves ? <dt>CVE:</dt> : null }
                                 { cves ? <dd>{cves}</dd> : null }
+                                { secSeverity ? <dt>{_("Severity:")}</dt> : null }
+                                { secSeverity ? <dd id="severity">{secSeverity}</dd> : null }
```

Comment:
This means that there are multiple elements with id "severity", no?  I think that is not well defined.  A class should work just as well for the tests.

Reply:
Well spotted!